extensions/amp-consent/amp-consent.md

```diff
@@ -228,5 +228,15 @@ AMP may support customizing blocking behaviors in the future. Because of this, t
 
 Individual components may override this behavior to provide more specialized handling. Please refer to each component's documentation for details.
 
+## Integrations and availablity 
+The table below lists the vendors and components that are integrated with amp-consent
+
+| Integration   | Prod Availability| Documentation|Ready For Testing
+| ------------- |------------------| -----| -----|
+| DoubleClick & AdSense Integration      | 05/10/18 | [Link](https://support.google.com/dfp_premium/answer/7678538) |Yes|
+| AMP IMA Video Integration   |  05/15/18  |   ||
+| AMP Geo |  05/10/18      |  [Link](https://ampbyexample.com/user_consent/geolocation-based_consent_flow/) |Yes|
```

Comment:
I am not sure about this. `<amp-geo>` doesn't integrate with `<amp-consent>`. It's `<amp-consent>` that integrate with `<amp-geo>` ....
`<amp-geo>` can be blocked by `<amp-consent>` just like any other components. TBH, I think we should not let `<amp-consent>` ever block `<amp-geo>`...